snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/AllSnomedApiTests.java

```diff
@@ -32,13 +32,24 @@
 import com.b2international.snowowl.snomed.core.issue.IssueSO2503RemoteJobDynamicMappingFix;
 import com.b2international.snowowl.snomed.core.request.ConceptSearchRequestSnomedTest;
 import com.b2international.snowowl.snomed.core.request.MemberSearchRequestSnomedTest;
+import com.b2international.snowowl.snomed.core.request.SnomedConceptMapSearchRequestTest;
 import com.b2international.snowowl.snomed.core.rest.branches.SnomedBranchingApiTest;
 import com.b2international.snowowl.snomed.core.rest.branches.SnomedMergeApiTest;
 import com.b2international.snowowl.snomed.core.rest.branches.SnomedMergeConflictTest;
 import com.b2international.snowowl.snomed.core.rest.branches.SnomedReviewApiTest;
 import com.b2international.snowowl.snomed.core.rest.classification.SnomedClassificationApiTest;
 import com.b2international.snowowl.snomed.core.rest.compare.SnomedMapTypeReferenceSetCompareTest;
-import com.b2international.snowowl.snomed.core.rest.components.*;
```

Comment:
Restore this single import statement and configure your IDE to use `*` imports above `10` import statements.